sematic/ui/src/hooks/httpHooks.ts

```diff
@@ -77,4 +77,31 @@ export function useHttpClient(): HttpClient {
         fetch: fetchCallback,
         cancel
     };
-}
\ No newline at end of file
+}
+
+/**
+ * Mock series of server side responses
+ * 
+ * @param data The series of HTTP Response JOSN
+ * @param interval delay before a request is fulfilled.
+ * @returns 
+ */
+export function useMockHttpClient<T>(data: Array<T> = [], interval: number) {
```

Comment:
This was introduced when I didn't have the backend endpoint available. I still think it's useful for future work. So let's keep it if possible.

Reply:
Can you move it to a new unit test file please?

Reply:
Hi @tscurtu. Thanks for the suggestion. I moved the file as you suggested, then found it misleading. As a React "hook", this utility is supposed to use with production code temporarily. It is, at this time, not designed for unit test purposes. 

I have yet to figure out how to use it in the unit test, have you?

Reply:
Sounds good, I renamed it to `useDebuggingHttpClient` now.